crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -82,6 +82,12 @@ pub struct Circle {
 }
 impl Primitive2d for Circle {}
 
+impl Default for Circle {
+    fn default() -> Self {
```

Comment:
Document the default values in a doc comment on the impl: they're a pain to discover otherwise.